### ISSUE TITLE
Unable to achieve the accuracy stated in the report

### ISSUE THREAD

**Issue by TheShy-Dream** [gchochla/VAuLT#6]
I replicated the experiment using README.MD and found that the accuracy of bertweet did not meet the reported effect

bertweet-base,twitter2015(train;dev),vilt-b32-mlm,False_0
![image](https://user-images.githubusercontent.com/58178926/233815507-6284b42b-f898-4346-9940-3d245fa85337.png)
bertweet-base,twitter(train;dev),vilt-b32-mlm,False_0
![image](https://user-images.githubusercontent.com/58178926/233815521-61eac0cd-8c54-4ba1-bc75-1243ffd6cd54.png)
bert-base-uncased,twitter2015(train;dev),vilt-b32-mlm,False_0
![image](https://user-images.githubusercontent.com/58178926/233815550-8a64ff9f-188b-4921-bce0-44d7b0ece44d.png)

**Comment by TheShy-Dream** [comment]
Can you give me some guidance on adjustments? Thank you very much

**Comment by gchochla** [comment]
I just ran the experiments in `scripts/test_results.sh` again, and I am getting virtually the same results as in the paper (here are the logs: [logs.zip](https://github.com/gchochla/VAuLT/files/11336474/results.zip)). Make sure you're on the latest version of the repo, have followed the installation instructions, and run the experiments from the aforementioned script to use the correct hyperparameters. FYI, my python version is `3.7.4`. Let me know if the problem persists, or if you already did follow all the provided instructions.

**Comment by gchochla** [comment]
@TheShy-Dream any updates on this? Or should I just close the issue?

**Comment by TheShy-Dream** [comment]
收到，辛苦了，祝万事如意，笑口常开，更上一层楼。
